src/AspNet.Security.OAuth.GitLab/GitLabAuthenticationExtensions.cs

```diff
@@ -0,0 +1,68 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authentication;
+using AspNet.Security.OAuth.GitLab;
+using JetBrains.Annotations;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    /// <summary>
+    /// Extension methods to add GitLab authentication capabilities to an HTTP application pipeline.
+    /// </summary>
+    public static class GitLabAuthenticationExtensions
+    {
+        /// <summary>
+        /// Adds <see cref="GitLabAuthenticationHandler"/> to the specified
+        /// <see cref="AuthenticationBuilder"/>, which enables Discord authentication capabilities.
+        /// </summary>
+        /// <param name="builder">The authentication builder.</param>
+        /// <returns>A reference to this instance after the operation has completed.</returns>
+        public static AuthenticationBuilder AddGitLab([NotNull] this AuthenticationBuilder builder)
+            => builder.AddGitLab(GitLabAuthenticationDefaults.AuthenticationScheme, _ => { });
+
```

Comment:
Please follow the code style of the other providers and use braces.

Reply:
reverted to brackets